packages/@cdklabs/cdk-cicd-wrapper/src/resource-providers/ComplianceBucketProvider.ts

```diff
@@ -35,6 +35,8 @@ export class ComplianceBucketConfigProvider implements IResourceProvider {
   provide(context: ResourceContext): any {
     const { account, region } = context.environment;
 
+    const vpcProvider = context.get(GlobalResources.VPC)!;
```

Comment:
why did you need to do this change? Please check and get the latest version of the code from main as seems you been working on an old release